superduper/misc/special_dicts.py

```diff
@@ -486,3 +486,22 @@ def recurse(data):
 
     recurse(data)
     return found_items
+
+
+class ArgumentDefaultDict(dict):
```

Comment:
Useful for several things.

Reply:
I am not sure if this can be replaced by `collections.defaultdict`.